transmart-api-server/grails-app/conf/application.yml

```diff
@@ -23,6 +23,14 @@ endpoints:
     enabled: false
     jmx:
         enabled: true
+    health:
```

Comment:
It looks like `management.security.enabled` and `endpoints.health.sensitive` detect what behavior to be for unauthenticated or authenticated users https://docs.spring.io/spring-boot/docs/1.5.3.RELEASE/reference/html/production-ready-monitoring.html#production-ready-health-access-restrictions
I assume that now we have:
`management.security.enabled=true`
`endpoints.health.sensitive=false`
Maybe it makes sense to make it explicit here what state of flags are and why it's important.

Reply:
there is some issue with `endpoints.health.sensitive`, changing it does not affect the behavior at all, so I would rather not add it here. But regarding `endpoints.health.sensitive` you're right. Apparently the defaults differ between spring-boot-actuator version, at least according to the documentation. I'll add it here.